src/store/tests/tests-reducer-factory.ts

```diff
@@ -10,6 +10,8 @@ import { TestCategory } from '@dvsa/mes-test-schema/category-definitions/common/
 import { CatBUniqueTypes } from '@dvsa/mes-test-schema/categories/B';
 
 import { testsCatBReducer } from './tests.cat-b.reducer';
+import { testsCatDReducer } from '@store/tests/tests.cat-d.reducer';
+import { CatDUniqueTypes } from '@dvsa/mes-test-schema/categories/D';
```

Comment:
this should error with eslint due to path imports needing to be ordered

Reply:
updated thanks